mongocrypt/src/ctx.rs

```diff
@@ -355,6 +355,8 @@ pub struct Ctx {
     inner: OwnedPtr<sys::mongocrypt_ctx_t>,
 }
 
+unsafe impl Send for Ctx {}
```

Comment:
For future readers, it might be worth mentioning why this is safe (you could just link to my comment I guess, not sure if that's susceptible to link rot though) and also why implementing `Sync` would be unsafe. Ditto for `KmsScope`. No need for me to re-review once you've added the comments.

Reply:
Done.